consensus/blake3pow/consensus.go

```diff
@@ -468,7 +468,7 @@ func (blake3pow *Blake3pow) CalcDifficulty(chain consensus.ChainHeaderReader, pa
 
 	if chain.IsGenesisHash(parent.Hash()) {
 		// Divide the parent difficulty by the number of slices running at the time of expansion
-		return new(big.Int).Div(parent.Difficulty(), big.NewInt(int64((blake3pow.NodeLocation().Region()+1)*(blake3pow.NodeLocation().Zone()+1))))
+		return parent.Difficulty()
```

Comment:
This is out of spec. Why aren't you ratio-metrically dividing it down? fwiw you guys still need to approve the spec...

Reply:
its only a test change, as we are running 3x3 directly, dividing doesnt work on startup, so just moved this back to what it was, the calculation is not right also, needs to be corrected once we finish the 3x3 test

Reply:
ok, please create an issue to come back and resolve this later.

Reply:
https://github.com/dominant-strategies/go-quai/issues/1675
created one and being tracked on board